rasa/nlu/tokenizers/tokenizer.py

```diff
@@ -61,6 +61,10 @@ def __lt__(self, other: Any) -> bool:
             other.lemma,
         )
 
+    def __repr__(self) -> Text:
+        return f"<Token object value='{self.text}' start={self.start} end={self.end} \
+        at {hex(id(self))}>"
+
 
 class Tokenizer(Component):
```

Comment:
```suggestion
class Tokenizer(Component):
    """Base class for tokenizers."""

```

Reply:
@ErickGiffoni please accept this change to resolve the code quality error. 
I ran the checks locally and the linter will also ask you to remove the empty lines after the docstrings of the class methods. 
These changes should allow the tests to pass (hopefully) 👍

Reply:
If you want to check code quality locally before committing you can run the following commands:
```
make formatter
make types
make lint
```

Reply:
Nice ! Thanks !